prow/pod-utils/decorate/podspec.go

```diff
@@ -593,6 +594,15 @@ func decorate(spec *coreapi.PodSpec, pj *prowapi.ProwJob, rawEnv map[string]stri
 	// TODO(fejta): we should pass around volume names rather than forcing particular mount paths.
 
 	rawEnv[artifactsEnv] = artifactsPath
+	gcsConfig := pj.Spec.DecorationConfig.GCSConfiguration
+	rawEnv[gcsArtifactsDirEnv] = path.Join(
+		gcsConfig.Prefix,
+		gcsConfig.Bucket,
+		"logs",
```

Comment:
For e.g. presubmits path is different than "logs".

Could you use:
https://github.com/kubernetes/test-infra/blob/24c8505d7bd2a59849f6a51fcd03c1239de190ed/prow/gcsupload/run.go#L143
like here:
https://github.com/kubernetes/test-infra/blob/master/prow/pjutil/pjutil.go#L278

Reply:
Done.